shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/domain/platform/VersionCode.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.blisskmmic.domain.platform
+
+interface VersionCode {
+    val versionCode: String
```

Comment:
There should be a line break 
```suggestion
interface VersionCode {

    val versionCode: String
```

Reply:
@blyscuit @minhnimble For kotlin, it is preferred to have a line break after class: https://nimblehq.co/compass/development/code-conventions/kotlin/#blank-line 🤔